go/providers.rst

```diff
@@ -204,6 +204,10 @@ method. In general, only rules_go should need to build or handle these.
 +--------------------------------+-----------------------------------------------------------------+
 | The exposed cc headers for these sources.                                                        |
 +--------------------------------+-----------------------------------------------------------------+
+| :param:`cc_info`               | :type:`CcInfo`                                                  |
++--------------------------------+-----------------------------------------------------------------+
+| The result of merging the ``CcInfo``s of all `deps` and `cdeps`                                  |
```

Comment:
Out of curiosity, is this autogenerated? If not is there any way to declare the outputs of our rules that would result in Stardoc being able to pick this up?

Reply:
I added this manually. I think that only the rules and extra docs are generated automatically, everything else is handwritten. In fact, I think that Stardoc can only produce Markdown, not RST.